### PR TITLE
[OSDEV-914] Updating the approved claim auto-reply message

### PR DESCRIPTION
**Pull request by vrwOAR**
Updating the text & html files of the claim_facility_approval email files to advise claimants of the editing behavior that will revoke their claim.

### CODE REVIEW

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
Changes look good to me, but there is one more file that needs to be updated: the `RELEASE-NOTES.md` file.
Since these changes will not be included in the `1.10` release, please add a new chapter for version `1.11` to the `RELEASE-NOTES.md` file. You can find the template for the new version in the `RELEASE-NOTES-TEMPLATE.md` file.
After adding the new chapter, please update the "What's New" section with your updates. You can refer to examples of how this section is filled for other release versions.
I would write something like this:
[OSDEV-914](https://opensupplyhub.atlassian.net/browse/OSDEV-914) - Update the approved claim auto-reply message to inform claimants of their editing parameters. This ensures that claimants are aware and less likely to edit the name and address beyond the approved claim values.

File: src/django/api/templates/mail/claim_facility_approval_body.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ Here's the reason for the approval:
 {{ approval_reason }}
 {% endif %}
 
-You are approved! You can now complete your profile, adding business information including production capabilities, certifications, MOQs and more. Visit opensupplyhub.org/claimed to view your approved facility and complete your profile.
+You are approved! You can now complete your profile, adding business information including production capabilities, certifications, MOQs and more. The "Facility name (English language)" & "Address" fields cannot be edited. Doing so will result in the claim being revoked. Visit opensupplyhub.org/claimed to view your approved facility and complete your profile.
```
[the remark this answers is not in the document]
Comment (vladsha-dev) [reply src/django/api/templates/mail/claim_facility_approval_body.txt:15]: I am not sure that it is possible in the `.txt` file...

[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply src/django/api/templates/mail/claim_facility_approval_body.txt:15]: Yes, thank you for noticing that.

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — DISMISSED** [review]
Approved

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved

**Pull request merged**